src/components/ElevatedButton/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+interface ElevatedButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  children: React.ReactNode;
+  isFullSize: boolean;
```

Comment:
I am whether it'll be more appropriate if we name this prop `isFullWidth`?

Reply:
@tyrro make sense, let me update it 🙏

Reply:
@tyrro resolved in [a31c486](https://github.com/manh-t/react-survey/pull/42/commits/a31c486e6aabdb861be574b7d76fd7ec5686a78d)